Item.cs

```diff
@@ -9,6 +9,79 @@ public class Item
         public override string ToString()
         {
             return this.Name + ", " + this.SellIn + ", " + this.Quality;
-        }  
+        }
+
+        public void UpdateQuality()
+        {
+            if (Name != "Aged Brie" && Name != "Backstage passes to a TAFKAL80ETC concert")
+                {
+                    if (Quality > 0)
+                    {
+                        if (Name != "Sulfuras, Hand of Ragnaros")
+                        {
+                            Quality = Quality - 1;
+                        }
+                    }
+                }
+                else
+                {
+                    if (Quality < 50)
+                    {
+                        Quality = Quality + 1;
+
+                        if (Name == "Backstage passes to a TAFKAL80ETC concert")
+                        {
+                            if (SellIn < 11)
+                            {
+                                if (Quality < 50)
+                                {
+                                    Quality = Quality + 1;
+                                }
+                            }
+
+                            if (SellIn < 6)
+                            {
+                                if (Quality < 50)
+                                {
+                                    Quality = Quality + 1;
+                                }
+                            }
+                        }
+                    }
+                }
+
+                if (Name != "Sulfuras, Hand of Ragnaros")
+                {
+                    SellIn = SellIn - 1;
+                }
+
+                if (SellIn < 0)
+                {
+                    if (Name != "Aged Brie")
```

Comment:
You can use switch statement on the name

Reply:
have done in the next pull request